ui-v2/src/components/dashboard/flow-runs-accordion/flow-runs-accordion-content.tsx

```diff
@@ -57,6 +67,26 @@ export function FlowRunsAccordionContent({
 	const flowRuns = data?.results ?? [];
 	const totalPages = data?.pages ?? 1;
 
+	// Prefetch next page on hover
+	const prefetchNextPage = useCallback(() => {
+		if (page < totalPages) {
+			const nextPageFilter = buildFilterForPage(page + 1);
+			void queryClient.prefetchQuery(
+				buildPaginateFlowRunsQuery(nextPageFilter, 30_000),
+			);
+		}
+	}, [page, totalPages, buildFilterForPage, queryClient]);
+
+	// Prefetch previous page on hover
+	const prefetchPreviousPage = useCallback(() => {
+		if (page > 1) {
+			const prevPageFilter = buildFilterForPage(page - 1);
+			void queryClient.prefetchQuery(
+				buildPaginateFlowRunsQuery(prevPageFilter, 30_000),
+			);
+		}
+	}, [page, buildFilterForPage, queryClient]);
```

Comment:
Add a prefetch for the task run counts too because the flow run card depends on that query.